src/ImportDefinitionsBundle/ProcessManager/ProcessManagerListener.php

```diff
@@ -103,4 +104,37 @@ public function onFinishedEvent(ImportDefinitionEvent $event)
     {
         $this->processLogger->info($this->process, ImportDefinitionsReport::EVENT_FINISHED.$event->getSubject());
     }
-}
\ No newline at end of file
+
+    /**
+     * @param ImportDefinitionEvent $event
+     */
+    public function onArtifactEvent(ImportDefinitionEvent $event)
+    {
+        // dispatches it via the logger to self::onProcessLogEvent()
+        $this->processLogger->info($this->process, ImportDefinitionsReport::EVENT_ARTIFACT.$event->getSubject(), [
+            'artifact' => $event->getSubject(),
+        ]);
+    }
+
+    /**
+     * @param ImportDefinitionEvent $event
+     */
+    public function onProcessLogEvent(ProcessLogEvent $event)
```

Comment:
This can probably end up in ProcessManager instead? It waits for the `artifact`, it doesn't care where it came from.

Reply:
yes and no actually. I think its still good that the artifact generation is logged.

Reply:
no wait, you are right. There is no need to log that again, PM can do that.